tensorboard/components/experimental/plugin_util/core-host-impl.ts

```diff
@@ -0,0 +1,32 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+/**
+ * Implements core plugin APIs.
+ */
+tb_plugin.host.listen('experimental.GetURLPluginData', (context) => {
+  if (!context) {
```

Comment:
I am really forgetful. Would it be possible to leave a comment on when this can happen?

Reply:
Never mind. The comment on `context` already covers this.